src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -30,9 +35,20 @@ internal JsonClassInfo GetOrAddClass(Type classType)
             return result;
         }
 
+        /// <summary>
+        /// The <see cref="JsonReaderOptions"/> used when deserializing an object or value type.
+        /// </summary>
         public JsonReaderOptions ReaderOptions { get; set; }
+
+        /// <summary>
+        /// The <see cref="JsonWriterOptions"/> used when serializing an object or value type.
+        /// </summary>
         public JsonWriterOptions WriterOptions { get; set; }
 
+        /// <summary>
+        /// The default buffer size used when creating temporary buffers while serializing or deserializing.
```

Comment:
It might be worth documenting the actual default buffer size value, especially since we are exposing it anyway.

Reply:
Done